### ISSUE TITLE
tf.keras.applications.EfficientNetBXs already contain a rescaling layer to preprocess input. It's confusing!

### ISSUE THREAD

**Issue by shelldonhu** [tensorflow/tensorflow#48412]
https://github.com/tensorflow/tensorflow/blob/85c8b2a817f95a3e979ecd1ed95bff1dc1335cff/tensorflow/python/keras/applications/efficientnet.py#L735-L737
`tf.keras.applications.efficientnet.preprocess_input `does nothing at all! 
`tf.keras.applications.EfficientNetBXs` already contain a rescaling layer to preprocess input. It's confusing!
`tf.keras.applications.EfficientNetBXs` and `tf.keras.applications.efficientnet.preprocess_input`'s design and behavior are different from other tf.keras.applications modules.

**Comment by ymodak** [comment]
Please take a look at this https://github.com/tensorflow/tensorflow/issues/45243#issuecomment-740845533 it applies to efficientnet as well.

**Comment by UsharaniPagadala** [comment]
@shelldonhu  Could you please fill the issue template

In order to expedite the trouble-shooting process, could you please provide the following details

OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
Mobile device (e.g. iPhone 8, Pixel 2, Samsung Galaxy) if the issue happens on mobile device:
TensorFlow installed from (source or binary):
TensorFlow version:
Python version:
Installed using virtualenv? pip? conda?:
Bazel version (if compiling from source):
GCC/Compiler version (if compiling from source):
CUDA/cuDNN version:
GPU model and memory

and the exact sequence of commands / steps that you executed before running into the problem. Thanks!

**Comment by shelldonhu** [comment]
OS Platform: Linux Ubuntu 16.04
Distribution: Tensorflow 2.3.0

 
![image](https://user-images.githubusercontent.com/16998373/114114452-774e9880-9913-11eb-886a-bc958313dbc8.png)

`tf.keras.applications.EfficientNetBX` already contains rescaling and normalization layers, the inputs pixel values will be scaled from [0, 255] to [-1, 1].

**Comment by ymodak** [comment]
The [`preprocess_input`](https://www.tensorflow.org/api_docs/python/tf/keras/applications/efficientnet/preprocess_input?version=nightly) docs are updated in nightly versions.
